backend/app/data_processing/gesture.py

```diff
@@ -81,3 +81,21 @@ def convert_gesture_to_audio(gesture, gesture_param):
     for pair in sonified_gesture:
         audio_samples.extend(generate_sine_wave_with_envelope(pair[0], pair[1]))
     return audio_samples
+
+
+def get_instrument_sliders(gestures, gesture_param):
```

Comment:
This is great as a start: the higher the overall x/y values of the gesture, the higher pitch we end up with as the sample.

I'd suggest we can preserve more of the character of each gesture in each sample, though: what if each sample, rather than being a continuous pitch modulated in frequency (or some other parameter) over time in response to the changing coordinates?